xapian-letor/tests/Makefile.am

```diff
@@ -81,14 +82,46 @@ EXTRA_DIST = collate-test generate-api_generated api_all.h api_collated.h \
 EXTRA_DIST +=\
 	runtest.in \
 	valgrind.supp \
-	testdata/apitest_rset.txt \
-	testdata/apitest_phrase.txt \
 	testdata/apitest_allterms4.txt \
+	testdata/apitest_phrase.txt \
+	testdata/apitest_rset.txt \
+	testdata/empty_file.txt \
 	testdata/qrel.txt \
+	testdata/qrelone.txt \
+	testdata/qrelthree_correct.txt \
 	testdata/query.txt \
+	testdata/queryone.txt \
+	testdata/querythree.txt \
 	testdata/score_qrel.txt \
 	testdata/training_data.txt \
-	testdata/training_data_ndcg.txt
+	testdata/training_data_ndcg.txt \
+	testdata/training_data_one_document.txt \
+	testdata/training_data_three_correct.txt
+
+CLEANFILES +=\
+	err_output_listmle_1.txt \
+	err_output_listmle_2.txt \
+	err_output_listmle_3.txt \
+	err_output_ListNet_1.txt \
+	err_output_ListNet_2.txt \
+	err_output_ListNet_3.txt \
```

Comment:
Not a problem with this patch as such, but at some point it's probably worth renaming the `*_ListNet_*.txt` files to be `*_listnet_*.txt` so the naming is consistent with all the others files here (and with the naming of files in the codebase in general).

Reply:
fixed in a fixup commit.